### DISCUSSION TITLE
Spiral Linux alongside MX and Win10

### DISCUSSION THREAD

**Discussion by beggink** [SpiralLinux/SpiralLinux-project#137] [General]
I just installed Spiral Linux in addition to Win10 and MX on my computer, on a separate SSD (/dev/sda). Since Spiral doesn't show up in my boot menu yet, I tried to make /dev/sda2 bootable from MX. But that didn't work; fdisk just says "No partition is defined yet"! Did I miss anything?
Thanks in advance for any help!
- Bernd

**Comment by CicadaSeventeen** [comment]
Go into uefi boot menu

**Comment by geckolinux** [comment]
Hi there, please include a screenshot of the partitioning screen in Calamares when installing to that configuration.
